### PR TITLE
Time to use FRAGILE_MELEE: round two, SPEAR_FORKED

### PR DESCRIPTION
**Pull request by AlienZimogor**
The past change in the knifespear caused a violent reaction, and therefore, changing all the weapons IMMEDIATELY, would inevitably cause an even more negative reaction. So I'll slowly move the Overton window in that direction until we can fix all of the "earlygame spearkite problem" weapons with FRAGILE_MELEE.
It's time to fix REALLY "big sacred cow" of initially spears - wooden pole with three tied spikes. Compare to knife spear, this ultimate weapon even more usable, and can be better than most of initially 1-hex weapons beacuse of fine damage and block2. Current situation completely ruin not only battles; starting options with points and recipes is completely useless, such as tonfu.

### CODE REVIEW

**Comment by Firestorm01X2** [comment]
I recommend to use more "detailed" and carefull  descriptions for PR. With much less emotions.

>Compare to knife spear, this ultimate weapon even more usable, and can be better than most of initially 1-hex weapons beacuse of fine damage and block2. Current situation completely ruin not only battles; starting options with points and recipes is completely useless, such as tonfu.

This is good

>SHOW_NO_MERCY, kevin!

This IS NOT!

**Comment by Catalyst2114** [comment]
I can see where some people are coming from on the knife spear nerf even if I personally don’t see the point but there’s no need to be this cavalier or to turn this into a witch hunt. Nerfing all spear weapons with reach to have FRAGILE_MELEE just makes them all equal AKA how they were before the nerf. It won’t affect what weapon you’ll choose to go early game scavenging for since the resource requirements for them are incredibley low to begin with which is why people argued for the FRAGILE_MELEE tag and you could just make more. Let’s have some good Faith discussion here please.

**Comment by Catalyst2114** [comment]
Also why should a tonfa, an improvised weapon itself that was originally a mechanism in grinding grain and meant to be non lethal, be better than a trident type weapon whose class was specifically meant to disarm opponents. The tonfa was never meant to kill, the forked spear is and should do it well

**Comment by AlienZimogor** [comment]
Looks like after violent and high-tonal discusses in past PR, i lost my DISCRETIZATION METER. 
Excuse me.....

**Comment by 0be1isk** [comment]
It wasn't violent, it was a group of people who thoughtfully and somewhat politely broke down your broken logic because you have some vendetta against spears, yet completely miss the entire point of what spears are (and furthermore the simple notion of "if you don't like them, don't use then")

**Comment by 0be1isk** [comment]
I personally use bows, because they deal far more damage at farther ranges, where you "kit" the zombies to the max, and are fully silent. Yet you seem ok with gun and bow kiting but god forbid spears (which the whole purpose of IS to counter melee based opponents) have the same ability. Spears are negated by spitters, smokers, boomers, and any zombie with a ranged attack

**Comment by Catalyst2114** [comment]
It would be appreciated Zimogor, if you could provide us with a list of early game weapons that you think should be better than the knife spear and forked spear.

**Comment by Catalyst2114** [comment]
Honestly Obelisk with the ranged weapon rebalance, I don’t think I used a melee weapon at all in 180 game days. Melee weapons in general seem to fall back in comparison and they provide the most in kiting. Once you cleared out the town and can begin on your deathmobile, well that’s your melee weapon right there

**Comment by AlienZimogor** [comment]
> Also why should a tonfa, an improvised weapon itself that was originally a mechanism in grinding grain and meant to be non lethal, be better than a trident type weapon whose class was specifically meant to disarm opponents. The tonfa was never meant to kill, the forked spear is and should do it well

And what i think about this situation:
1. Tonfa and forked spear have block2. This makes such nondescript weapon really choice, but without fragile and with reach, more damage... spear win.
2. On a start screen, if preconditions for skills are met, you can see the available recipes for crafting. Tonfa need "skills_required": [ "bashing", 3 ], forked spear need "skills_required": [ "survival", 1 ], its not a secret, what is a better choice. For expample, metal knuckles need "skills_required": ["bashing",1], but still need a saw_metal for craft. This very bad situation with craft, instruments and recipes can not be called good.
3. Last, and not least: forked_spear can be easily crafted without any time/skill investment. For train bash from the zero mark, need a go to forest and try to hit squirrels, ravens, and other small monsters(special CDDA-olympics discipline), wasting time and trigger a chance to encounter a wolfs, bears and ELKS, instead of, forked spear only need a crashed window... Not quite a good situation, do not you think?

**Comment by Catalyst2114** [comment]
Extremely good points Zimogor, but I think there's a better way to go about this than FRAGILE_MELEE which seems a bit disingenuous. The block2 for Forked Spear is a place holder for the disarm_block which is an issue in of itself which I'm fine with removing since the reach attack is good enough without an additional block. 

The tonfa is made from metal and is hard to make correctly without making it lesser quality IRL aka without adding a FRAGILE_MELEE tag to it. The forked spear is simple to make and anyone who works with wood knows how surprisingly sturdy shaped wood can be. 

To summarize, the tonfa is a relatively small improvised weapon while the forked spear is a long dedicated weapon. I'm willing to admit that the requirements are really easy and would be willing to compromise with additional requirements such as fabrication 2 or 3 which is a relatively mundane way to add balance by way of realism. I hope I haven't come off as adversarial but I just don't believe the tonfa is very good at all as a weapon.

**Comment by Catalyst2114** [comment]
Apologies ahead of time for bringing this time but throwing weapons are ridiculously strong, Keven Granade himself managed to hold off a horde with a pile of rocks in game to prove it. Yet we as a community more or less ignore them because we know this. What's the point in nerfing a fan favorite even if I don't use it when other people love it. The old adage "Don't give orders you know will be disobeyed" applies here. Plenty of controversy has erupted over this and only serves to divide. I don't care either way but come on, let's not start a civil war here guys

**Comment by AlienZimogor** [comment]
Oh, i think, it is obviviously for all, i mean tonfa_wood as a start weapon recipe. Not steel tonfa.

> It would be appreciated Zimogor, if you could provide us with a list of early game weapons that you think should be better than the knife spear and forked spear.
What weapons should(have to?) be better than forked and knife spear? For example...
1. Makeshift maul(homewrecker?)
2. Makeshift crowbar
3. Staff
4. Big (ogre) bat
5. Armature
And so on simple weapons, which can deal REAL damage, and easy(in game) to craft. 

> The block2 for Forked Spear is a place holder for the disarm_block which is an issue in of itself which I'm fine with removing since the reach attack is good enough without an additional block.
"There is nothing more permanent than temporary". At least, block2 settled for years in forked spear.

> and would be willing to compromise with additional requirements such as fabrication 2 or 3
Sometimes I thought about it, but a broken crafting system that raises the skill very quickly and without much time investment only spoils everything by simple recipes.

**Comment by Catalyst2114** [comment]
Oh in that class just drop the requirements for the tonfa from 3 to 1, add fabrication 2 to forked spear. I don't think the crowbar or staff should be better than a forked spear. A staff is just a long stick, and the makeshift crowbar is just a bent pipe. It makes sense for the early game skills to raise very quickly because it's low numbers.

**Comment by Catalyst2114** [comment]
I agree with you that it's too easy to make, but we should just add reasonable requirements to it. It's no fun to remake the same weapon multiple times which is why some people hate repairing clothing. The solution to the clothing was to make better, more durable clothes and I think the solution here is to add more requirements to it. The knife spear is the weakest spear and as a the first newbie weapon can survive a FRAGILE_MELEE tag, but let's not add it to all the spears when the issue is how easy it is to craft.

**Comment by Firestorm01X2** [comment]
Just do not forget that game already can calculate fragility based on weapon material. So do not go overboard with flag. Especially for solid objects.

**Comment by Catalyst2114** [comment]
Correct me if I'm wrong firestorm, but doesn't the knife spear require a basic repair kit to repair it? It's scarce enough that it's not a day 1 get and damaged weapons have lower damage prompting the player to make a new knife spear anyways. Instead of adding an artificial tag to it, can't we just let the natural course take it's course? No one should be using it for long anyways. it's a pretty bad weapon.

**Comment by Firestorm01X2** [comment]
You can disamvle it and reassemble. Also you can just create a lotbof spares

**Comment by Catalyst2114** [comment]
Doesn't disassembling damaged items have a chance to not return all of your components though? Yeah, we already make spares so why do we need the fragile tag

**Comment by kevingranade** [comment]
The rationale for this weapon being fragile is the same as for the knife spear, the crafting recipe literally consists of lashing a prong or prongs to a stick, this is not a robust way of attaching the blades.  This is reflected in both the materials for the recipe and the very low skill requirements.

This PR is not the only valid solution to this issue, but it is A valid solution.

I'd be fine with a non-fragile improved knife spear with higher crafting requirements to reflect that the blade is seated in the shaft and properly bound in place.  That doesn't really work for a forked spear as far as I can tell, because the requirement to seat the blades seems to conflict with the requirement to have multiple blades.

**Comment by mlangsdorf** [comment]
The current forked spear looks to be some spikes hammered, duct taped, or tied onto a shaft. There could be an upgraded trident made with a forge that had a proper, socketed head with three prongs. If the steel spear is the hand-forged upgrade of the knife spear, this trident would be the hand-forged upgrade to the forked spear.

**Comment by smolbird** [comment]
@kevingranade I'd recommend at this point closing the PR if the author can't moderate their tone.

You closing my issue for a perceived attitude and letting this person get away with such blatantly insulting attitude is hypocritical.

**Comment by smolbird** [comment]
Notable examples:

`Do you really think that no one understands this dull circus with a plugging through pits, bushes, windows and traps?`

`But we'd rather do the tables and still not notice the obvious for YEARS.`

`And after this all entire community can't fix this REAL BIG hole?`

`After this, speaking about "no one weapon use that backups before"- just a trolling.`

`It's time to fix REALLY "big sacred cow" of initially spears - wooden pole with three tied spikes. `

`SHOW_NO_MERCY, kevin!`

**Comment by Lorith** [comment]
In case this goes though anyways, preemptive un-nerf mod: https://cdn.discordapp.com/attachments/365907426270117888/457194828191629312/Un_Nerf_Spears.7z

Seriously, as mentioned numerous times in numerous places, spears are the main choice due to poor earlygame melee interaction even when trying to be safe.  Make biting harder for lone zombies, and/or deep bites easier to clean, and there will be a large drop in spear usage -- most weapons do more damage and are faster already.  Spears are fine, it is other melee that is the problem, mainly due to deep bites.  A lone person trying to bite you is not difficult to stop IRL, if you accept being hit while stopping them.  In CDDA that is not an option, and deep bites can be unavoidably fatal even if you instantly switch to sprinting around trying to find drugs to fix them.  One zombie touching you once is potential for unavoidable death.  THIS is why spears are the primary choice, not because they are horribly overpowered and need nerfing.

We do not need to nerf everything that gives you a chance of survival that isn't RNG based.

**Comment by Xhuis** [comment]
While the earlygame has its own problems, it's its own nut to crack overall. This PR is centralized in making another weapon generally nonviable overall, for the sake of what at a glance appears to be spite, not a desire to address the problem in a way that keeps the weapons fun to use.

In the current environment of the game, I maintain that reducing the speed of reach attacks, or the weapons in general, is more than enough to keep them viable but make them less strong in the part of the game that they overshadow right now.

**Comment by smolbird** [comment]
> Additionally, (opinion ahead!), this isn't a change made to preserve balance. It's a bad-faith PR made very clearly because its author dislikes spears and wants to spite people who does. None of his comments in this PR or the knife spear PR have been anything less than childish towards those who disagree with him, and I don't feel like that's the kind of attitude we should encourage.

This needs restating, yes. It also warrants requesting the feedback of @Coolthulhu and @Rivet-the-Zombie because so far no developers have stepped forward to give an appropriate response to a deliberately offensive and disruptive attitude, and in fact their bad-faith PRs are actually being approved by the only dev that is participating.

This sends the wrong kind of message to the community.

**Comment by 1skandar** [comment]
And yet again pointless. Now I’ll just craft a wooden spear and then a copper spear a bit later. This is stupid. You’re making the beginning game more harder for inexperienced players and more tedious for experienced players without addressing the underlying issues at all. And your attitude is just piss poor on top of it all

**Comment by 0be1isk** [comment]
Indeed. This guys attitude is extremely spiteful. Like I understand the idea and somewhat support it, but not in such a half assed way. Just because you got one of your half baked ideas into the masters doesn't mean that's a green light to continue spite everyone further.

I complete understand the need for "fragile" items. But it's either all or none. Why tf should a spear break every 20 hits yet my little glass bottle have unlimited durability?

**Comment by kevingranade** [comment]
> There could be an upgraded trident made with a forge that had a proper, socketed head with three prongs.

I agree, I was thinking a "better but still improvised trident", but a "proper" trident is certainly possible.
There's this kind of thing: https://www.google.com/search?q=primitive+fishing+spear
But that just highlights the fact that it's made to grab the target, not deal a lot of damage.
I'm pretty sceptical that a multi-headed weapon is a reasonable part of the progression, I'm not aware of any examples of such used for real combat.

Thinking on that a bit more, a better primitive spear can have several beneficial properties:
* More reach
* More damage per hit from a sharper or wider blade.
* Overall lighter for faster attacks.
* Crossbar to prevent lunges.

> @kevingranade I'd recommend at this point closing the PR if the author can't moderate their tone.

They did in fact moderate their tone, the very first reply called them out on it and they edited their post.
Then they apologized, and everything since then has been very matter-of-fact.
If you're talking about the previous PR, it presented a change I had already decided to merge in a previous PR after reading *that* discussion, but that had been reverted, so I merged it without reading the discussion carefully.

If I had I would have seen *this* gem, "This is the most astoundingly lazy, half-baked, and narrow-minded change I've seen so far.".  Park your fucking outrage dude, you're lucky I haven't banned you already.  Consider that a warning.

> Why tf should a spear break every 20 hits yet my little glass bottle can whack a hulk without ever shattering?

PRs are welcome.

**Comment by 1skandar** [comment]
I’m still going to side with this being a silly thing. You’re adding fragile melee to things that are *designed* to be weapons, improvised or not and not things you *could* use as melee weapons but probably shouldn’t. This is wildly unbalanced and does crap all to address the underlying issues.

**Comment by smolbird** [comment]
> Park your fucking outrage dude

That isn't any less impolite.

**Comment by DemAvalon** [comment]
I only found this on a makeshift trident video:[https://www.youtube.com/watch?v=qDr89PEsZWI](url)

An alternative would be to remove the forked spear and replace it with a sturdy makeshift spear (needs better name) with identical stats minus the supposed fragility, like this one:[https://www.youtube.com/watch?v=EzfGLKiIEhU](url)

Note tough the spear in the video is meant as a prop and made of aluminum, I see no reason why it could not be made using scrap steel (albeit taking much longer)

**Comment by Xhuis** [comment]
The fragile tag makes weapons four times as likely to take damage on a hit. While, yes, fastening a spike to a stick isn't the most effective weapon, it's still certainly *doable*, especially if it's fastened tightly with duct tape and the like. Fragile is intended for weapons made of weak materials like glass, or of things like cheap, thin replica metal on a replica sword.

Forked spears in particular require hammering, implying that the spikes are more firmly seated, and so is especially silly. If we're just going to give spears fragile because they're spears, why not give steel spears fragile, too? Qiangs? Sure, they're solid, tempered steel and weapons of war, but they're *spears!*

I think there are ultimately more interesting and engaging ways to handle the problem of reach attacks without making the weapons near-useless. Reducing speed, increasing stamina cost, or adding increased missed chance are all suggestions I've heard that make sense and keep the weapons viable. As it stands, this is removing viability from weapons solely to make them nonviable.

**Comment by smolbird** [comment]
> laugh emjoi

Now I see why not many people like working with Kevin. He is genuinely a disrespectful hypocrite.

**Comment by NotFuji** [comment]
> The fragile tag makes weapons four times as likely to take damage on a hit.

Did the math.  For an average new-game survivor, it's actually somewhere between 40-50x more likely the weapon gets damaged.

**Comment by kevingranade** [comment]
> Now I see why not many people like working with Kevin. He is genuinely a disrespectful hypocrite.

Can you possibly not see the multiple layers of hypocrisy you're operating on right now?  A *new contributor* appears with a PR, and you immediately respond with an insulting and pointless response, then you demand their PR be closed on the basis of "insulting" comments, none of which are in any way insulting, then you join in on accusing said new contributor of posting a PR in bad faith with no evidence "for the community".
Then I call you out on your hypocrisy, but give you a warning instead of a ban (my bad), and you reply with a snide comment.

Any of that sinking in?

Sorry for all this unpleasantness @AlienZimogor

**Comment by Xhuis** [comment]
While @smolbird is being blunt and rather disrespectful, I don't think it invalidates the complaints raised against @AlienZimogor, nor does their status as a new contributor make the change any less hamfisted. In fact, given that he made his account on GitHub and, on the same day, opened his knife spear PR with that dismissive tone he used up until this point, I'd say that makes his case *weaker*, not stronger.

I realize I might be singing to the wind here, but I do think it's important we keep civil and view the change from a mechanical standpoint without letting anger debase us to name-calling. There's no need for all that. Whether or not Kevin or smolbird are poor to work with is a matter of opinion and not relevant to this PR, which is a very important change for setting precedent going forward and I believe should be treated seriously. :>

Forgive my insistence on this. This is far from the worst vitriol I've seen on a GitHub community (never going back to that) but I have no desire to see free discussion and shared opinions in a comment section fall apart because some people started fighting in them. I've witnessed so many changes elsewhere merged or closed in bad faith or too quickly due to anger between parties, and it's rather frustrating.

**Comment by victorsegall** [comment]
FRAGILE_MELEE should be used, and should also be improved. Doing it, regardless of when its done, will require a new look at the variety of weapon recipes it affects, not only the weapons themselves.

#24018 and #24028 are very sensible responses to the fact that FRAGILE_MELEE is now in play. They address the weapon items, not their recipes.

Now we need variant recipes for the affected FRAGILE_MELEE weapons. The variant recipes would produce improved versions of the appropriate fragile weapons. Here's an off the cuff example (I'm not prepared; just have the idea).

The original:

```
{
    "type": "recipe",
    "result": "spear_forked",
    "category": "CC_WEAPON",
    "subcategory": "CSC_WEAPON_PIERCING",
    "skill_used": "fabrication",
    "skills_required": [ "survival", 1 ],
    "difficulty": 2,
    "time": 2600,
    "autolearn": true,
    "//": "Hammer to kinda shape the spikes for stable attachment--no forge used",
    "qualities": [ { "id": "HAMMER", "level": 1 } ],
    "//": "Extra duct tape/string needed to attach the spikes; rag/leather for grip.",
    "components": [
      [ [ "stick", 1 ], [ "broom", 1 ], [ "2x4", 1 ], [ "pool_cue", 1 ] ],
      [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
      [ [ "cordage", 2, "LIST" ], [ "duct_tape", 75 ] ],
      [ [ "spike", 3 ] ]
    ]
  },
```

An improved mid-tier variant:

```
{
    "type": "recipe",
    "result": "spear_forked_welded",
    "category": "CC_WEAPON",
    "subcategory": "CSC_WEAPON_PIERCING",
    "skill_used": "fabrication",
    "skills_required": [ "survival", 1 ],
    "difficulty": 3,
    "time": 12000,
    "autolearn": true,
    "//": "Hammer to kinda shape the spikes for stable attachment--no forge used",
    "qualities": [ { "id": "HAMMER", "level": 1 } ],
    "//": "Welder used to reinforce joins between spikes",
    "qualities" : [ { "id" : "GLARE", "level" : 2 } ],
    "tools": [
    [
      [ "oxy_torch", 4 ],
      ["welder", 20],
      ["welder_crude", 30],
    ]
    "//": "Extra duct tape/string needed to attach the spikes; rag/leather for grip.",
    "components": [
      [ [ "stick", 1 ], [ "broom", 1 ], [ "2x4", 1 ], [ "pool_cue", 1 ] ], [ "mop", 1 ] ],
      [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
      [ [ "cordage", 2, "LIST" ], [ "duct_tape", 75 ] ],
      [ [ "spike", 3 ] ]
    ]
  },
```


An top-tier variant based on spear_steel:

```
{
    "type": "recipe",
    "result": "spear_forked_forged",
    "category": "CC_WEAPON",
    "subcategory": "CSC_WEAPON_PIERCING",
    "skill_used": "fabrication",
    "skills_required": [ "survival", 1 ],
    "difficulty": 4,
    "time": 60000,
    "autolearn": true,
    "using": [ [ "forging_standard", 1 ], [ "steel_standard", 2 ], [ "filament", 100 ] ],
    "qualities": [
      { "id": "ANVIL", "level": 3 },
      { "id": "HAMMER", "level": 3 },
      { "id": "CHISEL", "level": 3 }
    ],
    "tools": [
      [ [ "tongs", -1 ] ],
      [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ]
    ],
    "components": [
      [ [ "2x4", 3 ], [ "stick", 6 ] ],
      [ [ "rag", 2 ], [ "felt_patch", 2 ], [ "leather", 2 ], [ "fur", 2 ] ]
    ]
  },
```

**Comment by kevingranade** [comment]
Consider these properties when making variants:
> Thinking on that a bit more, a better primitive spear can have several beneficial properties:
* More reach
* More damage per hit from a sharper or wider blade.
*  Overall lighter for faster attacks.
*  Crossbar to prevent lunges.

Someone correct me if I'm wrong, but I'm unaware of any serious weapons that are forked on order to increase damage.  Tridents and fishing spears are forked to increase the chance of stabbing a fish.

In short, please don't make new forked spears, but improved spears are fine.

**Comment by DracoGriffin** [comment]
>Now we need variant recipes for the affected FRAGILE_MELEE weapons. The variant recipes would produce improved versions of the appropriate fragile weapons. Here's an off the cuff example (I'm not prepared; just have the idea).

Yes, I think that's a better approach for the survivor to concocting more advanced designs with better stability and control and the like. The suggestions from kevin are also good ideas, although the enhanced reach may not be as feasible (unless it's a high-end crafting option, making a spear-type larger isn't always ideal) and the crossbar could be made into some form of technique -- or perhaps a new flag that prevents lunges from being successful in damage/knocking over target (would require coding).

**Comment by Lorith** [comment]
> The suggestions from kevin are also good ideas, although the enhanced reach may not be as feasible (unless it's a high-end crafting option, making a spear-type larger isn't always ideal)

I would have to disagree on the extra reach as well, spears are typically designed for certain ranges, so extending the range would require a large rework of the spear, basically leading to a new item anyway.  The other options all sound good though, along with using a more durable binding method, which was the original idea, I believe.

**Comment by victorsegall** [comment]
> using a more durable binding method, which was the original idea, I believe.

Correct. The three tiered recipe idea assumed (and I should have explained) that #24028 code will go in before this could work. The product of the original forked spear recipe has breakage opportunities on the spikes and the binding. The product of the new mid-tier recipe breaks only at the bindings, because the spikes are welded. The product of the new top-tier recipe does not receive the FRAGILE_MELEE flag.

Any other differences between the three conceived items are separately debatable, since here we're concerned with the gameplay surrounding weapon fragility, and are focusing on reach weapons as the most imbalanced for day zero characters.

**Comment by DracoGriffin** [comment]
>Those recipe mockups I posted are pointing toward a way to compromise for both sides.

Some weapons were intended (eventually, someone just got around to it before anyone else did with maybe a smidge of underlying motives) to be fragile -- they are crappy makeshift weapons.

However, the compromise should come from letting the system work properly *first* then re-examining the issue. Currently, no weapons have stuck in mechanics -- it was believed to have been in but it was removed 2 years and never refactored in. That's already a major change in combat once it's put back in place -- for both reach weapons and melee ones. Maybe reach attacks have higher inherent risk of being stuck in (even with increasing skill levels) compared to melee range weapons.

Secondly, bite chance seems quite high even in 1v1 instances, which may be resolved (there might be fine-tuning later) with #23955 so again, if a survivor is intelligent enough to lure a single zombie from a horde or an area, they shouldn't be punished for it. Likewise, it may also be smart to utilize traps and other things to weaken / soften up the target before engaging in melee with 0 combat skills.

Thirdly, we needed a better way to handle fragile weapons -- they shouldn't just rupture upon contact with a target, it should just be a steady progression to falling apart which #24028 seems to correct.

I'm sure there's something else I'm missing, but either way, I think this has been a great catalyst for moving things to where they should have been, and eye-opening for many people on how to contribute and become productive members of the community. I think your idea to have more advanced versions (basically evolving from junk taped together to actual designed weapons of warfare designed against targets that have no fear of death, or pain, or hunger, etc) is progress to a good system.

**Comment by DracoGriffin** [comment]
>We'd have fragile weapons with independent chances of either breaking or getting stuck. Then someone down the line is inspired to think it's logical that stuck causes a break. May as well delete reach weapons, sooner or later, if we aren't offering progression in the survivor's skill at crafting reach weapons.

You are presuming a bit much. If non-reach weapons at say, skill level 3 ([skill level 10 is considered "professional"](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Game-Balance#skill-system-scaling)), result in getting stuck 40% of the time (ignoring "level of stuckness" for now), reach weapons would result in getting stuck 45% of the time. And we could say at the end of the spectrum, skill level 10, getting stuck results in 5% of the time for non-reach, and 10% for reach (or 7.5%).

This is all independent of "fragile" weapons -- which is wholly unto itself. Makeshift / "fragile" weapons would just have inherent increases chances of getting stuck ([since they aren't designed as weapons of combat](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Weapons-Balance)) as well as more easily damaged from attacks (especially the less weapon skilled / higher strength).

**Comment by victorsegall** [comment]
> You are presuming a bit much.

I'm glad that this series of PR is a small bit of JSON, and annoyed that it is only that. Looking forward to seeing how it shakes out.

Rest of what you're saying makes sense. I don't know stuck mechanic well enough to comment, yet. If I'm presumptuous, it's from following these arguments surrounding that signal flare in the changelog last week.
